src/ExtensionsSample/Program.cs

```diff
@@ -28,25 +28,23 @@ public static async Task Main(string[] args)
 
             var builder = new HostBuilder()
                .UseEnvironment("Development")
-               .ConfigureWebJobsHost(o =>
+               .ConfigureWebJobs(webJobsBuilder =>
```

Comment:
We need a stable HostId workaround as I had below, for this sample to run. Please run and verify.

Reply:
Made the change to simplify that process in https://github.com/Azure/azure-webjobs-sdk/pull/1834

Still need to address the proper host ID provider logic to replace what we had before the migration.